DDRBot.py

```diff
@@ -1236,12 +1236,16 @@ async def show_screenshots(self, message):
             data = eal.get_jpeg_data_for(photo['file_path'])
             screenshot_files.append(discord.File(io.BytesIO(data), '%s-%s.jpg' % ((photo['game_name'], photo['last_play_date']))))
             archive_screenshot(message.author.id, '%s-%s.jpg' % (photo['game_name'], photo['last_play_date']), data)
-            if 'dance' in photo['game_name'].lower():
-                await self.db_add_queue.put(DBTaskWorkItem(message.author.id, '%s-%s.jpg' % (photo['game_name'], photo['last_play_date']), photo['last_play_date']))
-            if 'beatmania' in photo['game_name'].lower():
-                await self.db_add_queue.put(
-                    DBTaskWorkItem(message.author.id, '%s-%s.jpg' % (photo['game_name'], photo['last_play_date']),
-                                   photo['last_play_date'], game='iidx'))
+            if os.path.exists("DDR_GENIE_ON"):
+                if 'dance' in photo['game_name'].lower():
+                    await self.db_add_queue.put(DBTaskWorkItem(message.author.id, '%s-%s.jpg' % (photo['game_name'], photo['last_play_date']), photo['last_play_date']))
+                if 'beatmania' in photo['game_name'].lower():
+                    await self.db_add_queue.put(
+                        DBTaskWorkItem(message.author.id, '%s-%s.jpg' % (photo['game_name'], photo['last_play_date']),
+                                       photo['last_play_date'], game='iidx'))
+            else:
+                print("DDRGenie is not enabled so screenshot parsing is unavailiable")
```

Comment:
This actually got kind of annoying so let me know if you want me to just remove it and we can silently pass over the screenshot parsing.

Reply:
Yeah, I think just silently skipping parsing is fine, no need to log it here.

Reply:
Cool, fixed in: https://github.com/cyberkitsune/DDRBot/pull/5/commits/62c575817506dd1e558ee840f351c7c00e5de7ef